geometry_msgs/PoseStamped[] result
int32 id
string state
